openedx/core/lib/tests/test_xblock_utils.py

```diff
@@ -173,28 +173,6 @@ def test_replace_static_urls(self, course_id, anchor_tag):
         self.assertIsInstance(test_replace, Fragment)
         self.assertEqual(test_replace.content, anchor_tag)
 
-    @ddt.data('course_mongo', 'course_split')
-    def test_grade_histogram(self, course_id):
```

Comment:
wrong quote.  :)  That applied to the rationale for the skiptest above.

I think you meant:
"The reason you are getting those errors is because the environment settings for CMS and LMS are different. For example, 'courseware' is listed in INSTALLED_APPS in lms/envs/common.py, but NOT in cms/envs/common.py. This is on purpose. And so, one would think that tests that require the courseware_studentmodule should not belong in the openedx folder and should be moved to lms/lib, for example."